TACC/tup-ui
# fix(tup-cms): TUP-340 cms nav align / show search & login

## Overview

- Align CMS nav to the left.
- Show portal search, but not login (until a custom app provides one).

## Related

- [TUP-340](https://jira.tacc.utexas.edu/browse/TUP-340)
- requires https://github.com/TACC/Core-CMS/pull/557

## Changes

- set `INCLUDES_CORE_PORTAL` to `False`
- set `INCLUDES_PORTAL_NAV` to `True`
- set `INCLUDES_SEARCH_BAR` to `True`

## Testing

2. Open (local test) https://localhost:8000 **or** (remote test) https://dev.tup.tacc.utexas.edu.
    - Window must be 1200px or wider.
3. Verify:
    - CMS nav is left-aligned.
    - Search bar is right-aligned.
    - Header does **not** have portal nav.
        - Even when turned on, without Portal, nothing loads, because of fallback template.﹡
    - Search bar does **not** have a white border near it (as would normally separate search bar and portal nav).

<sub>﹡ But, if custom app was present serving a view at the correct URL, its nav should (I think) load because its URL would be checked loads before `urls.py`.</sub>

## UI

| Local |
| - |
| ![TUP-340 Local Test](https://user-images.githubusercontent.com/62723358/196545096-4fe88b86-9279-4b3e-8e08-756a7e609a1e.png) |
